rotkehlchen/api/rest.py

```diff
@@ -4603,6 +4610,19 @@ def refresh_protocol_data(self, cache_protocol: ProtocolsWithCache) -> dict[str,
                         message=f'Failed to refresh {cache_protocol.name.lower()} cache due to: {e!s}',  # noqa: E501
                         status_code=HTTPStatus.CONFLICT,
                     )
+            case ProtocolsWithCache.ETH_WITHDRAWALS:
```

Comment:
Is this where only the cache is cleared via the API? If yes you can ignore my other question.